src/map/script.cpp

```diff
@@ -26939,6 +26940,22 @@ BUILDIN_FUNC(macro_detector) {
 	return SCRIPT_CMD_SUCCESS;
 }
 
+BUILDIN_FUNC(opentips){
+#if PACKETVER < 20151202
+	ShowError( "buildin_opentips: This command requires PACKETVER 20151202 or newer.\n" );
+	return SCRIPT_CMD_FAILURE;
+#else
+	map_session_data* sd;
+
+	if (!script_charid2sd(3, sd)) {
+		return SCRIPT_CMD_FAILURE;
+	}
+
+	clif_ui_open( *sd, OUT_UI_TIP, script_getnum(st, 2) );
```

Comment:
Is there an ID range we need to check or is that done in the clif/client side?

Reply:
Not that I know. Unknown ID on client just opens a random tip.